### ISSUE TITLE
Update Doku about connection Automotiv readers.

### ISSUE THREAD

**Issue by morawekj** [kivijakola/hitager#13]
I do a lot of work to connect Renault card readers to hitager. I get lot of read errors, the fix is to connect Mode PIN 5 from PCF7991AT to ground to switch of the SPI filtering that limits the data rate there. Sometimes you just move a 0 Ohm resistors, other readers require som trace cut an rewiring. After that, is working fine.

**Comment by josefe17** [comment]
Cheers! I wish I had read you before charring an AW1102 (285916556R) board an hour ago due to a bad probing trying  to guess why it wasn't working. Tomorrow I will try this If I don't burst the coil wires while soldering. 

Which OEM card reader are you working with? Also, have you used them with the 2 wire modification protocol plus the 12 V level shifter of just connecting to the SPI pins?

Regards,

**Comment by morawekj** [comment]
I use two different boards Renault Boards A2C53253093 and S120235201A, the bigger one have a 0 Ohm resistor directly behind the yellow C, that have to move to the PCF7991, to Ground Pin5, the other smaller 
![photo_2022-12-20 00 34 30](https://user-images.githubusercontent.com/3058377/208548453-593f81f6-9f61-47dd-ae70-fbc33760bd69.jpeg)
was cut traces and solder bridges. I remove every not needed component and solder the connection at the end of PCF7991 SPI lines.

**Comment by josefe17** [comment]
Nice! I will try to find these bigger boards as AW1102 is tiny and attached to the physical key ring (https://fccid.io/QI8-AW1102). The S120235201A is the OEM P/N 264381970188, but I haven't found anything related to the first one.

Thank you very much!

**Comment by morawekj** [comment]
I get another one for transform to Hitager
![20221220_143639](https://user-images.githubusercontent.com/3058377/208692572-f0fa9343-b996-49a4-a53b-be9ca7e1121e.jpg)
Removed Case, bare PCB
![20221220_144011](https://user-images.githubusercontent.com/3058377/208692718-6ae4e6c8-5087-4334-8646-ff31051ddb64.jpg)
I remove all not needed parts, you don't have to do this, only the resistors directly at the solder points. The picture at the connection points, also move the resistor to turn off SPI Filtering, the 12 V Parts are not usable, they limit the data rate.
<img width="811" alt="Bildschirmfoto 2022-12-20 um 15 36 32" src="https://user-images.githubusercontent.com/3058377/208693335-318406ee-5b86-4150-9296-a36abf5338df.png">

**Comment by tusker-tools** [comment]
@morawekj, regarding the "Move" instruction in your last picture: Do you mean to say that the resistor needs to be be moved to the empty positions inside the big red circle? This is not completely clear to me.

I'm just about to collect some modification instructions for different hardware in a wiki. So we need to describe everything as clear as possible.

**Comment by morawekj** [comment]
Yes, that is correct, the resistor have be moved in the empty posistion in the red circle.

**Comment by josefe17** [comment]
Basically, you have to ensure that pin 5 is grounded instead of connected to Vcc. in this case, this board offers the ability to select the level by soldering a 0 ohm jumper resistor to either Vcc or GND, so it's very easy. However, other boards have the pin tied to Vcc with a track below the chip, so you have to make it up.

The one I had from a Megane had a couple of vias, so I drilled one to cut the connection and pulled low with a soldered wire directly to the pins. And yes, I make it work (I will post some pictures later with the exact part number, bu it is the one with a side loading card and a couple of LEDs)

**Comment by tusker-tools** [comment]
Thanks to everybody for contributing. Pictures of how to modify specific readers are always welcome!

I created a [wiki](https://github.com/kivijakola/hitager/wiki) now where I also added the picture of @morawekj. Feel free to give feedback and add more info there.
